cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/store/ActionCreator.ts

```diff
@@ -15,58 +15,17 @@
  */
 
 import { getCurrentNamespace } from 'services/NamespaceStore';
-import { GLOBALS } from 'services/global-constants';
 import { MyPipelineApi } from 'api/pipeline';
 import Store, { Actions, SORT_ORDER } from 'components/PipelineList/DeployedPipelineView/store';
-import { objectQuery } from 'services/helpers';
-import { PROGRAM_STATUSES } from 'services/global-constants';
-import {
-  IPipeline,
-  IPipelineStatus,
-  IStatusMap,
-  IRunsCountMap,
-} from 'components/PipelineList/DeployedPipelineView/types';
-import orderBy from 'lodash/orderBy';
-import StatusMapper from 'services/StatusMapper';
-
-const ProgramType = {
-  [GLOBALS.etlDataPipeline]: 'Workflow',
-  [GLOBALS.etlDataStreams]: 'Spark',
-};
-
-interface IPipelineParams {
-  appId: string;
-  programType: string;
-  programId: string;
-}
-
-const DEFAULT_STATUS: IPipelineStatus = {
-  status: PROGRAM_STATUSES.DEPLOYED,
-  lastStarting: null,
-};
+import { IPipeline } from 'components/PipelineList/DeployedPipelineView/types';
 
 export function fetchPipelineList() {
-  const namespace = getCurrentNamespace();
-
-  const params = {
-    namespace,
-    artifactName: GLOBALS.etlPipelineTypes.join(','),
-  };
-
-  MyPipelineApi.list(params).subscribe((res: IPipeline[]) => {
-    const pipelines = orderBy(res, [(pipeline) => pipeline.name.toLowerCase()], ['asc']);
-
-    Store.dispatch({
-      type: Actions.setPipeline,
-      payload: {
-        pipelines,
-      },
-    });
-
-    fetchRunsInfo();
+  Store.dispatch({
+    type: Actions.setPipeline,
   });
 }
 
+// TODO this function is not using the results from graphql
```

Comment:
let's fix this in this PR, otherwise this will break the current functionality

Reply:
I believe the only change was to use the new interface. Is there any other changes that I need to do?

Reply:
that is not indicated in the comment

Reply:
I updated this function. Are there any changes that I need to make?